roles/kube-init/defaults/main.yml

```diff
@@ -2,3 +2,5 @@ kubectl_user: centos
 kubectl_group: centos
 kubectl_home: /home/centos
 artifacts_install: false
+ipv6_enabled: false
+control_plane_listen_all: false
```

Comment:
Awesome, this is exactly what I looking for to see that it'd be an option. +1

Reply:
Yea, I'm going to file a follow up issue to really clean all this up, but for now, it seems to work. I want to refactor this whole thing to make it easier to pass in your own configuration files.